content/security/helmet.md

```diff
@@ -15,7 +15,7 @@ $ npm i --save helmet
 Once the installation is complete, apply it as a global middleware.
 
 ```typescript
-import * as helmet from 'helmet';
+import helmet from 'helmet';
```

Comment:
Why the change? Using the newest version of `helmet` I'm able to use `import * as helmet from 'helmet'` without any warning or errors. Node version 14.5.0

Reply:
Also on the latest version of `helmet`, node version 14.15.3 and the latest TypeScript SDK. I'm getting a compiler error: 
![image](https://user-images.githubusercontent.com/13184132/103426236-3c242700-4bb8-11eb-8816-482781845a3f.png)

Reply:
Still not getting the same message to show up. 

Updated node to 14.15.3
Updated Typescript to 4.1.3
Updated helmet at 4.3.1
Using `import * as helmet from 'helmet'` without an issue. 

Same functionality with both `yarn` and `npm`

Reply:
I don't know why it's working on your side but errors on mine, but I'm sure this is a good change. `Helmet` exports a default in `index.ts` (see [Helmet index.ts](https://github.com/helmetjs/helmet/blob/main/index.ts) and this 
![image](https://user-images.githubusercontent.com/13184132/103426632-27955e00-4bbb-11eb-8b7e-d7549176a73d.png), it clearly shows that it's a default type)

Reply:
[Line 201](https://github.com/helmetjs/helmet/blob/main/index.ts#L201) `export = helmet`. There's no `default` there, and there's no default in the `d.ts`

Reply:
You _can_ do a `default` import (`import helmet from 'helmet'`) but it isn't neccessary

Reply:
Yes, that's correct. The [TypeScript docs](https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require) indicate that `export =` (in `helmet` index.ts) requires you to do `import helmet = require('helmet')`. That's the same as `import helmet from 'helmet'. Also see this [SO answer](https://stackoverflow.com/a/35706271), that confirms my claims.

Reply:
> You _can_ do a `default` import (`import helmet from 'helmet'`) but it isn't neccessary

I missed this one. Yes, you can, but the SO answer linked above says this is not recommended.

> Attempting to use `import * as express` and then invoking `express()` is always illegal according to the ES6 spec. In some runtime+transpilation environments this might happen to work anyway, but it might break at any point in the future without warning, which will make you sad.

Reply:
Interesting. I wonder if some of this comes down to what `target` you're configured for